Makefile

```diff
@@ -62,3 +71,18 @@ build: clean
 .PHONY: clean
 clean:
 	rm -rf dist
+
+
+# full clean
+.PHONY: full-clean
+full-clean:: local-uninstall clean
```

Comment:
Could you please elaborate a bit here?

* Why does removing the egg directory not redirect stderr. Do we need these redirects at all?
* What is the benefit of the or/`||`?
* Will there be no pycache entries in the tests directory?
* From running the tests locally, I have a `.coverage` file in my `tests` directory as well. Depending on the situation, there might even be multiple files in the working directory.